geode-dunit/src/main/java/org/apache/geode/test/greplogs/ExpectedStrings.java

```diff
@@ -59,6 +59,7 @@ public static List<Pattern> create(String type) {
     expected.add(Pattern.compile("Found expected warning"));
     expected.add(Pattern.compile("CacheClosedException: The cache is closed."));
     expected.add(Pattern.compile("Invoked MembershipNotifierHook"));
+    expected.add(Pattern.compile("java.io.IOException: Connection reset"));
```

Comment:
The ticket says that in JDK 17, the exception type is changed from `IOException` to `SocketException`. Should this be reflected here?

Reply:
Thanks. I made the expected pattern for connection reset more general to catch both